### ISSUE TITLE
JetBrainsMono Nerd Font not equals (!=) ligature is broken in Windows Terminal on Windows 11

### ISSUE THREAD

**Issue by vdrandom** [ryanoasis/nerd-fonts#1038]
#### 🗹 Requirements

- [x] I have searched the [issues](https://github.com/ryanoasis/nerd-fonts/issues) for my issue and found nothing related and/or helpful
- [x] I have searched the [FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting) for help
- [x] I have searched the [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki) for help

#### 🎯 JetBrainsMono Nerd Font not equals (!=) ligature is broken in Windows Terminal on Windows 11

Experienced behavior:
Some sort of deformed not equals ligature

Expected behavior:
Not equals ligature

Example symbols:
`!=`

#### 🔧 Your Setup

- _Which font are you using (e.g. `Anonymice Powerline Nerd Font Complete.ttf`)?_
   - _JetBrainsMono Nerd Font_
- _Where did you get the file from (download link, self patched, source downloaded from link...)_
   - Downloaded from your website, used chocolatey to install it
- _Which terminal emulator are you using (e.g. `iterm2`, `urxvt`, `gnome`, `konsole`)?_
   - Windows Terminal
- _Are you using OS X, Linux or Windows? And which specific version or distribution?_
   - Windows 11

#### ★ Screenshots (Optional)
How it looks like in Windows Terminal:
![image](https://user-images.githubusercontent.com/3367071/211173256-0f6f1634-bca3-494e-add9-55a28cbecf63.png)
How it is supposed to look like (WezTerm):
![image](https://user-images.githubusercontent.com/3367071/211173278-6d2a1b12-9f6d-492e-ba3e-70e4d50315c9.png)

**Comment by vdrandom** [comment]
Hopefully I've done everything correctly when reporting this.

Also I should note that JetBrains Mono does not have this issue, the ligature in the original font works properly.

**Comment by Finii** [comment]
Thanks for reporting.

> Also I should note that JetBrains Mono does not have this issue

Well, I just checked, official is still JetBrainsMono `v2.242`, and that DOES have the problem (see linked Issue at JetBrains [in #963](https://github.com/ryanoasis/nerd-fonts/issues/963#issuecomment-1275602789)).

Anyhow, we fixed it here. If you download from the repo directly it should work fine, I guess we did not release yet after the fix.

https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/JetBrainsMono
